wiki/Game_mode/osu!/id.md

```diff
@@ -0,0 +1,291 @@
+<!-- TODO: strange descriptions in many places and unhelpful order if you're trying to see what this game mode is all about. -->
+
+# osu! (mode permainan)
+
+*Untuk kegunaan lain, lihat [osu! (disambiguasi)](/wiki/Disambiguasi/osu!).*
+
+![Tampilan permainan osu!](/wiki/shared/osu-gameplay.jpg "Antarmuka osu!")
+
+Mode permainan pertama yang namanya didasarkan dari seri ouendan.
+
+## Apa itu "Ouendan"?
+
+*[Osu! Tatakae! Ouendan](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan "Wikipedia")* adalah permainan ritme untuk *Nintendo DS*. Alur permainan ini terdiri dari tiga elemen: mengetuk lingkaran pada layar sentuh, menyeret bola melintasi jalur dan memutar *spinner* dengan sangat cepat. Semua elemen ini diatur waktunya oleh (cover dari) lagu-lagu jepang populer, Ini akan terlihat seperti ini di *DS*:
+
+![Contoh alur permainan dari Osu! Tatakae! Ouendan di Nintendo DS](/wiki/shared/Ouendan.jpg "Gameplay example of Osu! Tatakae! Ouendan in Nintendo DS")
+
+Ceritanya bisa dilihat di layar atas, dan lingkarannya bisa dilihat di layar bawah. Setiap tahap pada dasarnya adalah cerita pribadi tentang seseorang yang bermasalah. Di situlah *Ouendan* (Skuad Penyorak) masuk. Dengan kekuatan magis dari pemandu sorak pria, pemain harus membantu orang yang bermasalah.
+
+## ![](/wiki/shared/mode/osu.png) Alur Permainan
+
+*Tutorial: [new beginnings (Sudah termasuk di dalam osu!)](https://osu.ppy.sh/beatmapsets/1011011).*
+
+Alur permainannya menekankan pada:
+
+- Adaptasi dan reaksi terhadap rangsangan (yaitu, objek yang dapat diketuk muncul di seluruh layar dengan kecepatan kilat).
+- Memahami ketukan lagu dan pola.
+- Kontrol perangkat masukan.
+- Waktu klik pada objek yang muncul di layar.
```

Comment:
```suggestion
- Timing pada kapan untuk mengklik pada objek yang muncul di layar.
```

Reply:
i think using 'pada' two times is not effective

Reply:
perhaps ```Timing pada kapan untuk mengklik objek yang muncul di layar.``` will do?